src/test/util/mod.rs

```diff
@@ -83,28 +85,46 @@ impl TestClient {
         ))
         .unwrap();
 
-        let build_info = RunCommand::new("test".into(), doc! { "buildInfo":  1 }, None).unwrap();
+        let mut server_version = None;
+        let mut server_parameters = None;
 
-        let response = client
-            .execute_operation_with_session(build_info, &mut session)
-            .await
-            .unwrap();
+        if collect_server_info {
+            let build_info =
+                RunCommand::new("test".into(), doc! { "buildInfo":  1 }, None).unwrap();
+
+            let response = client
+                .execute_operation_with_session(build_info, &mut session)
+                .await
+                .unwrap();
 
-        let info: BuildInfo = bson::from_bson(Bson::Document(response)).unwrap();
-        let server_version = info.version.split('-').next().unwrap();
-        let server_version = Version::parse(server_version).unwrap();
+            let info: BuildInfo = bson::from_bson(Bson::Document(response)).unwrap();
+            let server_version_str = info.version.split('-').next().unwrap();
+            server_version = Some(Version::parse(server_version_str).unwrap());
+
+            let get_parameters =
+                RunCommand::new("admin".into(), doc! { "getParameter": "*" }, None).unwrap();
+
+            if let Ok(response) = client
```

Comment:
Is it okay to ignore an error here? Might be better to unwrap the response to catch some kind of failure.

Reply:
I specifically chose to ignore a failure for the time being (and probably should've added a comment explaining so) because the current AWS and X509 auth tests don't allow for running `getParameter` commands. Since those tests don't need to check parameters (yet), I decided to ignore errors. The result of this is that any unified spec test that has a `runOnRequirement` that includes `serverParameters` will be skipped since the parameters will always be unequal.

Reply:
Ah gotcha, thanks for explaining that!